docs/UserGuide.md

```diff
@@ -156,55 +156,58 @@ Examples:
 The deadline field will show an `OVERDUE` label if the deadline has passed and the client status is still `active`, so if the client has paid, and your business with them is finished, remember to set their client status to 'old' so the deadline field doesn't show `OVERDUE`.
 </div>
 
-### Locating Clients By Name: `find`
+### Locating Clients: `find`
 
-Finds persons in address book who match parameters specified. Values matched are case insensitive.
+Finds persons in client list who match parameters specified. Values matched are **case-insensitive**.
 
 Format: `find [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]… [ps/PROJECT_STATUS] [py/PAYMENT_STATUS] [cs/CLIENT_STATUS] [d/DEADLINE]`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-* Names only need to match the start of a word. e.g. `find n/Han` OR `find n/B`
-    matches `Hans Bo`
+* For finding by name, the search is case-insensitive. e.g `hans` will match `Hans`.
```

Comment:
Not just name, find with any parameter is case-insensitive

Reply:
I just realized there is already a line that states this; will just move it here.